blockchain/storage/batch.go

```diff
@@ -0,0 +1,98 @@
+package storage
+
+import (
+	"math/big"
+
+	"github.com/0xPolygon/polygon-edge/helper/common"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/umbracle/fastrlp"
+)
+
+type Batch interface {
+	Delete(key []byte)
+	Write() error
+	Put(k []byte, data []byte)
+}
+
+type BatchHelper struct {
```

Comment:
Rename it to `BatchHandler` maybe (and rename file accordingly to `batch_handler.go`) WDYT?